packages/core/package.json

```diff
@@ -248,7 +248,6 @@
     "uuid": "^9.0.0"
   },
   "devDependencies": {
-    "@keystone-6/core": "5.7.0",
```

Comment:
The package.json was self referencing the `@keystone-6/core` package. I remove this one as it seems this was mistakenly added.

Reply:
this is needed for the tests to run through. The scripts are self referencing `@keystone-6/core/package.json`. readding it